### ISSUE TITLE
limb regrowth or re attachment 

### ISSUE THREAD

**Issue by Hectorr13** [studio-minus/ppg-bugs-and-requests#1778]
there used to be a limb regrowth mod but the mod creator could not keep up because of updates to the game and the mod no longer works . so you should add a limb regrowth or re attachment so there is one and its vanilla . for example in the wire menu there should be a muscle wire to re attach limbs or even things like metal poles , knifes , guns , ect . and there should also be a limb regrowth syringe for any types of way a ragdoll would lose a limb , chainsaw , ripped , bomb , ect . 

thats all i got i just think this would be a cool idea

**Comment by DindinYT37** [comment]
Duplicate or Similar requests:
* #254
* #435
* #629
* #709
* #1612
* #1631 
* #1652
* #1739

Also, please use the template.

**Comment by SirAJ4895** [comment]
Dang, people REALLY wanna glue limbs back on

**Comment by Hectorr13** [comment]
> 

yes

**Comment by Drogo-man** [comment]
I never really got it, you can't do that irl.

**Comment by Hectorr13** [comment]
> 

Who said the games supposed to be realistic

**Comment by DindinYT37** [comment]
> Who said the games supposed to be realistic

I mean that's what most requests wish for, though this is still a sandbox game so it should be possible.

**Comment by Iamjusthereforthegames** [comment]
> > 
> 
> Who said the games supposed to be realistic

Well, they've just made an update that made the gore more realistic, and just about every update has been doing that.

**Comment by DindinYT37** [comment]
> > Who said the games supposed to be realistic
> 
> Well, they've just made an update that made the gore more realistic, and just about every update has been doing that.

Yeah but we also have Laser Rifles, Aliens, Black Holes and Portals in this game, so being able to re-attach limbs certainly wouldn't be the most unrealistic thing we have.
